cmd/harmony/flags.go

```diff
@@ -1828,3 +1836,49 @@ func applyShardDataFlags(cmd *cobra.Command, cfg *harmonyconfig.HarmonyConfig) {
 		cfg.ShardData.CacheSize = cli.GetIntFlagValue(cmd, cacheSizeFlag)
 	}
 }
+
+var (
+	gasPriceBlocksFlag = cli.IntFlag{
+		Name:     "gasPrice.blocks",
```

Comment:
All of these need to be renamed similar to `gpo.blocks` per #4357

Reply:
Changed to 
```
--gpo.blocks
--gpo.percentile
--gpo.defaultprice
--gpo.maxprice
--gpo.txssampled 
```